### DISCUSSION TITLE
OperationalError: (2006, 'MySQL server has gone away')

### DISCUSSION THREAD

**Discussion by bm55** [WeblateOrg/weblate#9216] [General]
I am getting the database close issue with both MariaDB and Postgres DB. I have to restart the container to resolve it everyday. Any sugestion about what I can try? I tried to set the env PAPERLESS_DB_TIMEOUT but I get TypeError: 'timeout' is an invalid keyword argument for connect() error.

**Comment by nijel** [comment]
Disabling persistent connections might help.
